moha/utils.py

```diff
@@ -23,7 +23,8 @@ def convert_indices(N, *args):
     if len(args) == 4:
         # Check if indices are right
         for elem in args:
-            if not isinstance(elem, int) and not isinstance(elem, np.int64):
+            if not isinstance(elem, int) and not isinstance(elem, np.int64) \
```

Comment:
I'll change its to if type(elem) not in [int, np.int64, np.int32], because this line is getting too long